### PR TITLE
Made a new folder structer and implemented the constant colors

### PR DESCRIPTION
**Pull request by GruescuDanziel**
I have remade the folder structure and also implemented the colors into the respective place.

### CODE REVIEW

File: src/hooks/RoomApi.ts
Comment (alexgaita) [inline src/hooks/RoomApi.ts:1]: Api files should not be inside the hooks folder

**Comment by CatalinTiu** [comment]
Looks good

**Review by CatalinTiu — CHANGES_REQUESTED** [review]
Could you change "MettingRoom"  and "MettingRoomForm" folders' names to "MeetingRoom" and "MeetingRoomFolder"?

**Comment by GruescuDanziel** [comment]
Due to checking another branch, I had duplicate folders with the old names such as Components and GlobalTheme. That is the reason for the deletion.

**Review by CatalinTiu — APPROVED** [review]
Great!

**Comment by Liubzzz** [comment]
> Could you change "MettingRoom" and "MettingRoomForm" folders' names to "MeetingRoom" and "MeetingRoomFolder"?

good idea

**Pull request merged**